src/js/modules/infragistics.ui.editors.js

```diff
@@ -5176,14 +5173,14 @@
 
 					// Raise Warning level 2
 					this._sendNotification("warning",
-						$.ig.util.stringFormat($.ig.Editor.locale.maxValExceedSetErrMsg,
+						$.ig.util.stringFormat(this._getLocaleValue("maxValExceedSetErrMsg"),
 							this.options.maxValue));
 				} else if (!isNaN(this.options.minValue) && newValue < this.options.minValue) {
 					newValue = this.options.minValue;
 
 					// Raise Warning level 2
 					this._sendNotification("warning",
-						$.ig.util.stringFormat($.ig.Editor.locale.minValExceedSetErrMsg,
+						$.ig.util.stringFormat(this._getLocaleValue("maxValExceedSetErrMsg"),
```

Comment:
Please, check if you didn't change minValExceedSetErrMsg with maxValExceedSetErrMsg